.gitignore

```diff
@@ -34,6 +34,7 @@ test_results
 /doc/ui/portfolio
 
 /vendor/cmep
+/vendor/bundle
```

Comment:
My fault for putting the comment on the wrong side of the diff.

On Wed, Sep 16, 2015 at 12:43 PM, Bo Sun <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/nasa/earthdata-search/pull/15#discussion_r39653840>:
>
> > @@ -34,6 +34,7 @@ test_results
> >  /doc/ui/portfolio
> >
> >  /vendor/cmep
> > +/vendor/bundle
>
> It is failed due to another issue from a spec file. See commit 8061976
> <https://github.com/nasa/earthdata-search/commit/80619769ebd3c47be57a3e8939a9930e39e71bfd>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nasa/earthdata-search/pull/15/files#r39653840>.
>